WcaOnRails/app/models/competition.rb

```diff
@@ -222,6 +235,7 @@ def update_foreign_keys_when_id_changes
       CompetitionMedium.where(competitionId: id_was).update_all(competitionId: id)
       CompetitionDelegate.where(competition_id: id_was).update_all(competition_id: id)
       CompetitionOrganizer.where(competition_id: id_was).update_all(competition_id: id)
+      DelegateReport.where(competition_id: id_was).update_all(competition_id: id)
```

Comment:
Yeah, I was thinking something similar.

A self maintaining way is to hard code the classes and columns, and then have a test that compares that list with the schema.